tools/alerts/alert_registry.py

```diff
@@ -0,0 +1,119 @@
+import re
+from sre_constants import BRANCH
+from typing import Dict, Any, List, Union
+
+import urllib
+import os
+from alert_utils import create_issue, clear_alerts, fetch_alerts_filter, REPO_OWNER, TEST_INFRA_REPO_NAME, GRAPHQL_URL, UPDATE_ISSUE_URL, ISSUES_WITH_LABEL_QUERY,CREATE_ISSUE_URL, PYTORCH_ALERT_LABEL, update_issue
+ALERT_REGISTRY = {}
+
+PENDING = "pending"
+NEUTRAL = "neutral"
+SKIPPED = "skipped"
+SUCCESS = "success"
+FAILURE = "failure"
+CANCELED = "canceled"
+
+# rename this when these are ready
+# PYTORCH_ALERT_LABEL = "pytorch-alert"
```

Comment:
May be you could add a `--dry-run` mode here.  So the idea is that PR will use `--dry-run` mode for testing.  Then you could have `PYTORCH_ALERT_LABEL` set to `pytorch-alert-test` in dry-run and `pytorch-alert` otherwise.  So there is no need to comment stuffs out